mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt

```diff
@@ -126,6 +127,8 @@ class WidgetAdapter(
             widgets.forEach { w -> widgetsById[w.id] = w }
             notifyDataSetChanged()
         }
+
+        hasVisibleWidgets = widgets.any { widget -> isWidgetIncludingAllParentsVisible(widget) }
```

Comment:
Don't you also need to update this in `updateWidget()`?
I also think a getter method for this would be a bit more elegant.

Reply:
Done